Cargo.toml

```diff
@@ -82,18 +82,18 @@ relayer-amplifier-state = { git = "https://github.com/eigerco/axelar-relayer-cor
 
 # Solana Gateway
-axelar-solana-gateway = { git = "https://github.com/eigerco/solana-axelar.git", rev = "ee2b5b0", features = ["no-entrypoint"] }
-axelar-solana-its = { git = "https://github.com/eigerco/solana-axelar.git", rev = "ee2b5b0", features = ["no-entrypoint"] }
-axelar-solana-governance = { git = "https://github.com/eigerco/solana-axelar.git", rev = "ee2b5b0", features = ["no-entrypoint"] }
-gateway-event-stack = { git = "https://github.com/eigerco/solana-axelar.git", rev = "ee2b5b0" }
-axelar-solana-encoding = { git = "https://github.com/eigerco/solana-axelar.git", rev = "ee2b5b0" }
-axelar-solana-gas-service = { git = "https://github.com/eigerco/solana-axelar.git", rev = "ee2b5b0", features = ["no-entrypoint"] }
-axelar-solana-memo-program = { git = "https://github.com/eigerco/solana-axelar.git", rev = "ee2b5b0", features = ["no-entrypoint"] }
-axelar-solana-gateway-test-fixtures = { git = "https://github.com/eigerco/solana-axelar.git", rev = "ee2b5b0" }
-axelar-message-primitives = { git = "https://github.com/eigerco/solana-axelar.git", rev = "ee2b5b0" }
-axelar-executable = { git = "https://github.com/eigerco/solana-axelar.git", rev = "ee2b5b0" }
-its-instruction-builder = { git = "https://github.com/eigerco/solana-axelar.git", rev = "ee2b5b0" }
-interchain-token-transfer-gmp = { git = "https://github.com/eigerco/solana-axelar.git", rev = "ee2b5b0" }
+axelar-solana-gateway = { git = "ssh://git@github.com/eigerco/solana-axelar-internal.git", rev = "443377d", features = ["no-entrypoint"] }
+axelar-solana-its = { git = "ssh://git@github.com/eigerco/solana-axelar-internal.git", rev = "443377d", features = ["no-entrypoint"] }
+axelar-solana-governance = { git = "ssh://git@github.com/eigerco/solana-axelar-internal.git", rev = "443377d", features = ["no-entrypoint"] }
+gateway-event-stack = { git = "ssh://git@github.com/eigerco/solana-axelar-internal.git", rev = "443377d" }
+axelar-solana-encoding = { git = "ssh://git@github.com/eigerco/solana-axelar-internal.git", rev = "443377d" }
+axelar-solana-gas-service = { git = "ssh://git@github.com/eigerco/solana-axelar-internal.git", rev = "443377d", features = ["no-entrypoint"] }
+axelar-solana-memo-program = { git = "ssh://git@github.com/eigerco/solana-axelar-internal.git", rev = "443377d", features = ["no-entrypoint"] }
+axelar-solana-gateway-test-fixtures = { git = "ssh://git@github.com/eigerco/solana-axelar-internal.git", rev = "443377d" }
+axelar-message-primitives = { git = "ssh://git@github.com/eigerco/solana-axelar-internal.git", rev = "443377d" }
+axelar-executable = { git = "ssh://git@github.com/eigerco/solana-axelar-internal.git", rev = "443377d" }
+its-instruction-builder = { git = "ssh://git@github.com/eigerco/solana-axelar-internal.git", rev = "443377d" }
+interchain-token-transfer-gmp = { git = "ssh://git@github.com/eigerco/solana-axelar-internal.git", rev = "443377d" }
 
```

Comment:
This is being changes to internal because we have the latest gov module id change in the `next` branch, that is not pushed in the public repo.

Reply:
CI fails because of this of course ...

Reply:
I've also pushed the `next` branch to the public repo, you can push the latest `next` changes from the internal one to the public repo and use that one. Then we can merge as-is

Reply:
done